frontend/workflows/audit/src/index.tsx

```diff
@@ -25,13 +25,19 @@ const register = (): WorkflowConfiguration => {
         displayName: "Logs",
         description: "View audit log",
         component: AuditLog,
+        layoutProps: {
+          variant: "standard",
+        },
```

Comment:
This would be great! The only issue is leaving the layout variant undefined, and doing something like this:

```
layoutProps: {}
```

I think for that case setting it as `null` could work.